charts/hivemq-platform-operator/values.yaml

```diff
@@ -164,3 +164,32 @@ crd:
   # Waiting timeout for the CustomResourceDefinition to become ready once installed.
   # Format is based on the ISO-8601 Duration format.
   waitTimeout: PT10S
+
+# Monitoring configuration options
+monitoring:
+  # When enabled, deploys a ServiceMonitor resource for the Operator along with a Configmap for a Grafana dashboard.
+  enabled: false
+  # Dashboard configuration options.
+  dashboard:
+    # Creates a default HiveMQ Platform Operator Grafana dashboard ConfigMap.
+    create: true
+    # Namespace where to deploy the Grafana dashboard ConfigMap. Defaults to the namespace of the HiveMQ Platform Operator.
+    # namespace: monitoring
+    # The name of a Kubernetes ConfigMap with a custom HiveMQ Platform Operator Grafana dashboard to override the default dashboard.
+    # (also set create=false).
+    # Grafana dashboard ConfigMap content must be in JSON format along with the label `grafana_dashboard: "1"`
+    name: ""
```

Comment:
I don't understand the use-case of this config option. If we have `create=false`, why do we need to know an external ConfigMap name here? So far I only found one test that checks if the name is valid, but no further usage. What do we do with this when it's valid?

Reply:
Very well spotted! Removing as not needed.